bin/cheribsdtest/cheribsdtest.c

```diff
@@ -84,7 +84,9 @@
 #include "cheribsdtest.h"
 #include "cheribsdtest.h"
 
-#ifdef __mips__
+#if defined(__aarch64__)
+#define	TRAPNO_CHERI	0
```

Comment:
This is wrong and causes tests to fail; capability faults for loads/stores from a lower exception level are 0b100100 ie 36 (and if we have any tests that try to set an invalid PCC, those would be instruction aborts from a lower exception level i.e. 0b100000 == 32).